app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/itemlisting/ItemListingViewModel.kt

```diff
@@ -123,11 +119,7 @@ class ItemListingViewModel @Inject constructor(
             }
 
             is ItemListingAction.ItemClick -> {
-                handleItemClick(action)
-            }
-
-            is ItemListingAction.EditItemClick -> {
-                handleEditItemClick(action)
+                handleItemClick(action.authCode)
```

Comment:
Not a blocker for me, but more a discussion. @david-livefront on the password manager I know we have been avoiding unwrapping actions at this level and always passing the full action into the `handle*()` function. I am seeing there are somethings in the `when` that already kind of break away from some of those conventions like sending events, but curious if we want to keep that pattern up in Auth too.

Reply:
That is a good point, it is a little odd here that these are not actions though. I wonder if the `handle` moniker is the problem 🤔